jupyterlab_bigquery/src/components/query_editor/query_text_editor/query_text_editor.tsx

```diff
@@ -628,6 +631,10 @@ class QueryTextEditor extends React.Component<
         onClick={_ => {
           const query = this.editor.getValue();
           copy(query.trim());
+          this.props.openSnackbar({
+            message: 'Query copied',
+            autoHideDuration: 2000,
```

Comment:
Can you make this a const in a central location? Like `const AUTO_HIDE_DURATION = 2000`?

Reply:
Yes! I added a `const COPIED_AUTOHIDE_DURATION = 2000` in `snackbar.tsx` and moved `snackbar.tsx` to `src/shared/snackbar`